### ISSUE TITLE
Conda create does not find dependencies

### ISSUE THREAD

**Issue by quartox** [regro/conda-metachannel#22]
I am using a corporate mirror of the hosted metachannel, so if you can't reproduce then I need to deal with IT on my end.

I am just trying to create a conda environment with python and it is unable to find `pip` and `libcxx`. Adding `pip` to the channel url then says it can't find `wheel`. I looked at the `repodata.json` and confirmed that it isn't there.

Example:

`$ conda create --name test-meta --override-channels -c https://metachannel.conda-forge.org/conda-forge/python python=3.6`

Error:

```
Collecting package metadata: done
Solving environment: failed

PackagesNotFoundError: The following packages are not available from current channels:

  - python=3.6 -> libcxx[version='>=4.0.1']
  - python=3.6 -> pip

Current channels:

  - https://metachannel.conda-forge.org/conda-forge/python/osx-64
  - https://metachannel.conda-forge.org/conda-forge/python/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

**Comment by mariusvniekerk** [comment]
You may need to add pip as well to the packages required

**Comment by quartox** [comment]
I tried adding pip and it could not find wheel.

**Comment by quartox** [comment]
Everything works fine with I use it for `conda install` commands but it doesn't find dependencies when I use `conda create`. Again, this may be an corporate mirror issue and not a problem at all for the outside, but I don't have an easy time testing because my laptop is locked down (though I will try again tonight to see if I can get around that). If no one can reproduce than we should close this.

**Comment by quartox** [comment]
I was able to re-produce using the official url. `conda create` fails to find package dependencies, whereas `conda install` succeeds.

Might be intended and just something worth mentioning in README?

**Comment by mariusvniekerk** [comment]
@quartox I've fixed this in #24

**Comment by quartox** [comment]
It's working for me, thanks!
